CHANGELOG.md

```diff
@@ -86,10 +86,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
-- [Extensions] Moving Extensions APIs to protobuf serialization. ([#6960](https://github.com/opensearch-project/OpenSearch/pull/6960))
+- [Extensions] Moving Extensions APIs to support cross versions via protobuf. ([#7402](https://github.com/opensearch-project/OpenSearch/issues/7402))
```

Comment:
Can we add the PRs related to protobuf here, similar to [this](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.3.0.md?plain=1#L16)?

Reply:
I did like the approach we did for Searchable snapshots: https://github.com/opensearch-project/OpenSearch/issues/5087
See release notes: https://github.com/opensearch-project/OpenSearch/releases/tag/2.7.0 
It captures it cleanly. 
Adding more PR links just adds more and more numbers but not really adding value for readers. 
WDYT?

Reply:
Ah it's a meta issue. I confused it with this PR link. Linking the meta issue definitely makes more sense than linking multiple PRs.